### PR TITLE
refactor(frontend): common util to check ck-tokens ledgers

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Small refactoring to avoid repetition of code.

### CODE REVIEW

File: src/frontend/src/icp/utils/ic-send.utils.ts [hunk 1]
```diff
@@ -30,14 +31,19 @@ export const isBtcAddress = (address: BtcAddress | undefined): boolean => {
 export const invalidBtcAddress = (address: BtcAddress | undefined): boolean =>
 	!isBtcAddress(address);
 
+const isTokenCkLedger = (
```
Comment (peterpeterparker) [inline src/frontend/src/icp/utils/ic-send.utils.ts:34]: - `isTokenLedger`?

- Can you use an object as parameter?

Comment (peterpeterparker) [reply src/frontend/src/icp/utils/ic-send.utils.ts:34]: I prefer

```
export const isTokenCkBtcLedger = (token: Partial<IcToken>): boolean =>
	isTokenLedger({token, ledgerIds: CKBTC_LEDGER_CANISTER_IDS});
```

Comment (peterpeterparker) [reply src/frontend/src/icp/utils/ic-send.utils.ts:34]: Btw. if you are up to setup an eslint rules when you got a bit of time, that would be super.

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/utils/ic-send.utils.ts:34]: About this one? what would you put as rule?

Comment (peterpeterparker) [reply src/frontend/src/icp/utils/ic-send.utils.ts:34]: > Always pass an object unless one single parameter?

I would say so, what do you think?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/utils/ic-send.utils.ts]: I like it! Even because I didn't know what was the convention eheh

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/utils/ic-send.utils.ts]: In the meanwhile, corrected this one, have a look please

**Review by peterpeterparker — APPROVED** [review]
LGTm, thx

File: src/frontend/src/icp/utils/ic-send.utils.ts (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/utils/ic-send.utils.ts]: Done with PR:

- #2416

**Pull request merged**